monkestation/code/datums/diseases/advance/symptoms/necropolis.dm

```diff
@@ -0,0 +1,135 @@
+/datum/symptom/necroseed
+	name = "Necropolis Seed"
+	desc = "An infantile form of the root of Lavaland's tendrils. Forms a symbiotic bond with the host, making them stronger and hardier, at the cost of speed. Should the disease be cured, the host will be severely weakened"
+	stealth = 0
+	resistance = 5
+	stage_speed = -5
+	transmittable = -3
+	level = 8
+	base_message_chance = 5
+	severity = -1
+	symptom_delay_min = 1
+	symptom_delay_max = 1
+	var/tendrils = FALSE
+	var/chest = FALSE
+	var/fireproof = FALSE
+	threshold_descs = list(
+		"Stealth 7" = "Upon death, the host's soul will solidify into an unholy artifact, rendering them utterly unrevivable in the process.",
+		"Resistance 12" = "The area near the host roils with paralyzing tendrils.",
+		"Resistance 15" = "Host becomes Immune to heat and ash",
+
+	)
+	var/list/cached_tentacle_turfs
+	var/turf/last_location
+	var/tentacle_recheck_cooldown = 100
+
+/datum/symptom/necroseed/Start(datum/disease/advance/advanced_disease)
+	if(!..())
+		return
+	if(advanced_disease.totalResistance() >= 12)
+		tendrils = TRUE
+		if(advanced_disease.totalResistance() >= 15)
+			fireproof = TRUE
+	if(advanced_disease.totalStealth() >= 7)
+		chest = TRUE
+
+/datum/symptom/necroseed/Activate(datum/disease/advance/advanced_disease)
+	if(!..())
+		return
+	var/mob/living/carbon/Victim = advanced_disease.affected_mob
+	switch(advanced_disease.stage)
+		if(2)
+			if(prob(base_message_chance))
+				to_chat(Victim, "<span class='notice'>Your skin feels scaly.</span>")
+		if(3, 4)
+			if(prob(base_message_chance))
+				to_chat(Victim, "<span class='notice'>[pick("Your skin is hard.", "You feel stronger.", "You feel powerful.")]</span>")
+		if(5)
+			if(tendrils)
+				tendril(advanced_disease)
+			Victim.dna.species.brutemod = min(0.6, Victim.dna.species.brutemod)
+			Victim.dna.species.burnmod = min(0.6, Victim.dna.species.burnmod)
+			Victim.dna.species.heatmod = min(0.6, Victim.dna.species.heatmod)
+			Victim.add_movespeed_modifier(/datum/movespeed_modifier/necro_virus)
+			ADD_TRAIT(Victim, TRAIT_PIERCEIMMUNE, DISEASE_TRAIT)
+			if(fireproof)
+				ADD_TRAIT(Victim, TRAIT_RESISTHEAT, DISEASE_TRAIT)
+				ADD_TRAIT(Victim, TRAIT_RESISTHIGHPRESSURE, DISEASE_TRAIT)
+				ADD_TRAIT(Victim, TRAIT_LAVA_IMMUNE, DISEASE_TRAIT)
+				ADD_TRAIT(Victim, TRAIT_ASHSTORM_IMMUNE, DISEASE_TRAIT)
+				ADD_TRAIT(Victim, TRAIT_SNOWSTORM_IMMUNE, DISEASE_TRAIT)
```

Comment:
compatibility with snowstorm variant, thematically it makes perfect sense since its about thicc hide that slows down the user, noe that it does not give cold resistance.